samples/quickstart/quickstart.py

```diff
@@ -37,7 +37,7 @@ def run_quickstart():
     )
     print("\nProcessing video for label annotations:")
 
-    result = operation.result(timeout=120)
+    result = operation.result(timeout)
```

Comment:
This moves the value out of the snippet, which means people copy/pasting it from the website don't see it declared. Can we just leave it here and increase the value to 180?

Reply:
OK I will do that